src/nginxconfig/util/angular_backwards_compatibility.js

```diff
@@ -78,6 +78,8 @@ const domainMap = {
 
     https:                  ['https', 'https', oldBool],
     http2:                  ['https', 'http2', oldBool],
+    http3:                  ['https', 'http3', oldBool],
+    port_reuse:             ['https', 'portReuse', oldBool],
```

Comment:
```suggestion
```
These aren't needed as these options weren't present in the angular version of the tool :)